src/views/payment-sheet-views/card-view.js

```diff
@@ -114,6 +113,33 @@ CardView.prototype._sendRequestableEvent = function () {
   }
 };
 
+CardView.prototype._renderCardIcons = function () {
+  var overrides = this.merchantConfiguration.overrides;
+  var cardIcons = this.getElementById('card-view-icons');
+  var supportCardBrands;
+
+  cardIcons.innerHTML = cardIconHTML;
+
+  try {
+    supportCardBrands = overrides.fields.number.supportedCardBrands;
```

Comment:
should the name of this variable match what we're setting from overrides? Currently it's `supportCardBrands`, I'm wondering if it should be `supportedCardBrands`